bedrock/firefox/templates/firefox/new/batm/resist.html

```diff
@@ -1,13 +0,0 @@
-{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
They're only testing alternative headlines and according to Fabio that test isn't even planned to begin for a few weeks. It's possible these templates could come back as they are if the control is the winner, but if A or B is the winner we'd have to update these variants for the next headline test. We can leave them in place even though they're not getting traffic, they're not hurting anyone, I was just thought I'd clear out templates that aren't being used. 

I'm probably just piling more onto this already-too-complicated PR than I should. I'll put these back and we'll pretend it never happened :)